packages/dds/tree/src/schema-stored/Stored and View Schema.md

```diff
@@ -35,104 +34,116 @@ Places we can store stored schema information:
-    This forces all nodes of the same type to have these same schema: its not contextual, but easily handles open polymorphism (ex: a child thats allowed to be anything, as long as its in schema for its type).
--   In other schema: for example, a schema can apply specific schema to its children.
+-   At the root (ex: shared-tree, a subclass, or its configuration), which can be special cased to apply some rules, possibly recursively, to the root.
+-   Declaratively on nodes of a specific type: Nodes in the tree are typed, so similar to the above, the tree can be configured to apply specific schema based on type of the nodes.
+
+    This forces all nodes of the same type to have the same schema: it's not contextual but easily handles open polymorphism (ex: a child that's allowed to be anything as long as it's in schema for its type).
+-   In other schema: For example, a schema can apply specific schema to its children.
+
     This allows for `contextual schema` (under one parent the same type might have different rules compared to under another parent).
 
-Note that its possible to refer to schema in a way that's unambiguous, but code handing the data might not always have the schema.
-For example document could refer to a schema by its hash, or name in a append only namespace.
+Note that it's possible to refer to schema in a way that's unambiguous, but code handling the data might not always have the schema.
+For example, document could refer to a schema by its hash, or name in an append only namespace.
 This can have interesting implications for updates to new schema (ex: one client adds data using a schema shipped as code that another client does not have).
 Some of the options do not have this issue (inline, central repository (assuming you are ok going down if it goes down and schema are not cached), and in stored data).
 
-# What options to support.
+# Options to Support
 
-Thats a lot of options, and I think there are good reasons for all of them, so long term, we likely want all of them to be options to some extent.
-But I think we can pick a good subset for the now, and leave open the option of extending it in the future.
+That's a lot of options and I think there are good reasons for all of them. Long term, we likely want all of them implemented to some extent but I think we can pick a good subset for now and allow extension in the future.
 
 I think we want to at least:
 
 -   Make adoption of different constraint/schema systems optional and incremental: all of these different options should be opt in, and easy to add to and existing application if desired.
--   Provide a schema-on-read system (ex: schematize) that can give an application a nice interface to the data, including validation and error handling
+-   Provide a schema-on-read system (ex: schematize) that can give an application a nice interface to the data, including validation and error handling.
+
```

Comment:
This blank like breaks the formatting, right? This sentence is supposed to be part of the above bullet point.

Reply:
It does not, at least on my VSCode markdown preview. If it's indented properly, it keeps it under the same bullet but in a new paragraph.